README.md

```diff
@@ -48,15 +48,35 @@ Those are all Environment variables needed by the application:
 | TEST_CLIENT_REDIRECT_URI | The redirect uri of the test client                        | string |
 
 ## Example
-Start the server locally and then:
+Start the server locally:
 
 ``` sh
-curl "http://localhost:3000/oauth/authorize" \
--d client_id=example-client \
--d response_type=id_token \
--d redirect_uri=https%3A%2F%2Fclient.example.org%2Fcb \
--d scope=openid \
--d state=af0ifjsldkj \
--d nonce=n-0S6_WzA2Mj \
--b X-IO-Federation-Token=12345
+./start.sh
+```
+1. Add a new client:
+
+``` sh
+curl --location --request POST 'http://localhost:3000/connect/register' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+   "redirect_uris":
+     ["https://client.example.org/callback"],
+   "client_name": "This is a test client",
+   "grant_types": ["implicit"],
+   "response_types": ["id_token"],
+   "token_endpoint_auth_method": "none"
```

Comment:
Can we specify the scopes?

Reply:
Yep! With `scope` attribute. And should match the values provided in [provider.ts#L94](https://github.com/pagopa/io-openid-provider/blob/2b8ece52f981ac39270fd23cffe57cb9346163c8/src/oidcprovider/provider.ts#L94).

Example: 

``` sh
curl --location --request POST 'http://localhost:3000/connect/register' \
--header 'Content-Type: application/json' \
--data-raw '{
   "redirect_uris":
     ["https://client.example.org/callback"],
   "client_name": "This is a test client",
   "grant_types": ["implicit"],
   "response_types": ["id_token"],
   "token_endpoint_auth_method": "none",
   "scope": "openid"
}'
```

## Refs
1. https://datatracker.ietf.org/doc/html/rfc7591#section-2